source/_integrations/solax.markdown

```diff
@@ -39,6 +39,22 @@ template:
 
 {% endraw %}
 
+### Configuring the Energy Dashboard
+
+There are generally 3 sensors from your inverter that you need to configure in the energy dashboard:
+
+- the total consumption sensor (in kWh) for the "Grid Consumption"
+- the total feed-in sensor (in kWh) for the "Return to grid", i.e. the solar panel energy that you do not consume and return to the grid instead
+- the total on-grid yield sensor (in kWh) for the "Solar production"
+
+These are "running total" sensors, do not use the "daily" sensors.
```

Comment:
Why not? That is totally valid.

If you can select them on the energy dashboard it means they are compatible.

Reply:
Do you mean that it does not matter which one I use? I have both "daily" and "total" sensors - they both work in the energy dashboard?
If yes, then we should change this to clarify, indeed.

Reply:
Yes you can use them both